recipes/stempeg/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "stempeg" %}
+{% set version = "0.1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+    sha256: f2266ad32d4e4ff3a936279b62a48ba58e6ada32f0e5ee53f4968877c0148136
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - ffmpeg
+    - setuptools
```

Comment:
just to double-check, do you really need ``setuptools`` to run this project?

Reply:
I think we can remove it, right?
```suggestion
```

Reply:
Good question, I will check that (I am losing myself in the ocean of dependencies to adapt in Conda package in order to have our library available and kinda get confused sometimes ^^')

Reply:
Effectivly, `setuptools` is not a direct dependencies, only used by `setup.py` provided script. I remove it :)

Reply:
It seems it is a direct dependency, somewhere is importing ``pkg_resources`` which is distributed with ``setuptools`` :/

Reply:
https://github.com/faroit/stempeg/blob/256f460b176d7508b6d71a3607b4435fc3350efc/stempeg/__init__.py#L13

Reply:
Actually it is needed for `pkg_resources` module so I will restore it